state that the server may send a subset of fields by default (#1500)
json-api/json-api: _format/1.0/index.md

The value of the `fields` parameter **MUST** be a comma-separated (U+002C
COMMA, ",") list that refers to the name(s) of the fields to be returned.

> An empty value indicates that no fields should be returned.

Reviewer: 👍

If a client requests a restricted set of [fields] for a given resource type,
an endpoint **MUST NOT** include additional [fields] in resource objects of
that type in its response.

If a client does not specify the set of [fields] for a given resource type,

> the server **MAY** send a subset of the set of fields defined for the resource
> type; an empty set is acceptable.

Reviewer: ```suggestion
the server **MAY** send all fields, a subset of fields, or no fields for that resource type.
```
